package.json

```diff
@@ -18,12 +18,18 @@
     "format": "prettier --write \"{apps,packages}/**/*.{ts,tsx,md,css,scss}\""
   },
   "devDependencies": {
+    "@csstools/postcss-oklab-function": "4.0.8",
     "prettier": "3.5.0",
     "turbo": "^2.3.4"
   },
   "msw": {
     "workerDirectory": [
       "./apps/web/public"
     ]
+  },
+  "dependencies": {
+    "@evilmartians/harmony": "1.4.0",
+    "react-circular-progressbar": "2.2.0",
+    "react-hook-form": "7.54.2"
   }
```

Comment:
These should not be installed in the root-package but the individual apps/packages